WeakAuras/RegionTypes/AuraBar.lua

```diff
@@ -51,6 +51,25 @@ Private.regionPrototype.AddAlphaToDefault(default);
 local screenWidth, screenHeight = math.ceil(GetScreenWidth() / 20) * 20, math.ceil(GetScreenHeight() / 20) * 20;
 
 local properties = {
+  barTextureSource = {
```

Comment:
The names of the properties need to match the aura data's property names, that is these should be textureSource, textureInput and texture.

The reason for that is Condition.lua's GetBaseProperty, which on decativating a condition undoes the change by looking up the unmodified value in the aura's data.

E.g. this aura:
!WA:2!1v1xVnXrqCF1suHfbbMwutvujfvqTubLMgO)Lw8zSJnYXj98f(NO1zVBx7BlNVD1U75eN3WQsWZUFbQ8Z9j)qFTv8jyvu)eKpbv5dqPZU3rcGi5H4zgnZSZ8BM535uV8GY4Y4NU8ujhjE0DPyv07(mR8A96jjQ75i2otQXF)RN75)BHcfeJYnCZB)7Na0lmdLegXeRZOjQGQ1A7xZBxBk8ykKIYsC4N(v0xLHjUv2WFnxAiljASsq73NiKN8IICX)0vnItCttOkE5ujPlkqYebniOytWBUNmnGmKKO6K2RhD7zDRwPJF3o(v8837q3TU6z9ZgQk6G4wxqG486SETwTgl5K44MyzPXMynp6M6tgWJrJicVe0aIS0umjaEkFOQenQ1A96B0AsAsE1wIFAuOIoK4NPBE3)OWutZ1nKfZe3Ui83(a8qejO47aTkGc)2ECbRVGiLDyPIqYLwOWBnrsI7zbs9XghGevZI2XPiuasKkvGuKipumpcv8LNt33zk0AEK(qIL3(S652fudqHpQVGLMGZn0JjizgE688YkY2QUYieMT19YYsXjwBM)D(lW5ZFGwxiYbivxEUGNIoGG3lldhuYhNFgRfuQIbordT7tUva19ZYcDBc2A8X8p6nKCtw7gt6Jch1TxmJjIEM1HFjvQO9gPp2uHzfI4dgDBVw7A8p8OYcxqcPgyUOEob06gx2nBrnVzDMAd7Wb0K80KO4NQUGUZI)ykcdaoArFF(PEj0kBQ4atLxbeVFwExOWUhKiZ(IBBwcjZ0wmb(UceF8DZfYAVHqHget2m)rEPZPPnB3UMx3w1Q7Z)GJQvZ0C4lEuomGH7AwC28WcRdDhY88p5OIapcw7PHDvrW(zelgpkh7Y7rNN(o6lvu)PN)cj6VZIe6ROpU(Z0xv)5pw30r)f6JPxwFn9C6Vu)1w0v)n6VvFJfkOVPUI2DtD1zz9MxZvA4R)E9pOxzE9x58gw6sYfCtaGmiIq7hbKpEN7)weiFSTuQGKDdTuRoR6gZq46bkumCNFwVbPXkAjx4gp8SAVsEHXiP0i5kbmWiSN9oYfChBUCxQYTUf)uatqfSzVJGj4vrBhTlSvdaWACddMS0mAswvbALMGsOzIx3tQqcv9NrqsshLGK0xf92am0AxS54fCPRHAtija7awo24Nzlr3YDaIMux)twV1)SUREtDRGE0eQm61nV)HN2hSmUEXRELR96fonjAQWYkyPUqqna0kcdDf0fxx)Ws6GsUWBOkz5RYGXfkmXsEjPyINDaTukfpt0aVXYp4(r8Lxj77fvp42zMvVHD2Cdr(mzSBmDNDqcS3wMB(g3K9xphMzozex(zoX)4MgEXEOqYdRIG78K(UiXd3O5Lpu7YDSF(Wgw0uflCygf6758ew(ltXysIx7A3PMN7om2aNz9eaZnmcGd4I7NT9w3yYaebDQ6vRw7ZtXl1MSva0R9O9ljOjMetI2JKGGdYveimfwIIMXeMFTJVPnwZR5dwRTFLwtnJqQfjF)l6fgrcF011NyHcJhIeutcMeHmRd9jkVHO4usrryekPpbCx)e(PFb8SyZejSbPgng(IamLuJG5RihFwLGPOjV4BblbOnq7CM(5L2Ahwy6XLLlDlYWYd)N79)p

has a condition changing the texture on having a target, but the decativation of that condition doesn't work.

Reply:
fixed this and added `textureSource = "LSM"` to default table